changelog/unreleased/breaking-changes/1656.md

```diff
@@ -0,0 +1,8 @@
+VAST will not send layout information from incoming table slices
+to connecting sources anymore. Instead, only the layouts from schema
+files will be sent.
+
```

Comment:
This newline will cause this to be in two separate changelog entries.

Reply:
Also, here's a suggestion for rewording:

> Schemas are no longer implicitly shared between sources, i.e., an `import` process importing data with a custom schema will no longer affect other sources started at a later point in time. Schemas known to the VAST server process are still available to all `import` processes. We do not expect this change to have a real-world impact, but it could break setups where some sources have been installed on hosts without their own schema files, the VAST server did not have up-to-date schema files, and other sources were (ab)used to provide the latest type information.